### DISCUSSION TITLE
What about tests?

### DISCUSSION THREAD

**Discussion by teneko** [rajanadar/VaultSharp#264] [General]
No part of the project is tested. That could consider many to not use the library. Is there any ideas to implement tests, foremost basic integration tests?

**Comment by rajanadar** [comment]
hi @teneko 

I am not sure where you got this from, "No part of the project is tested."
That is incorrect and misinformed.

VaultSharp is tested using actual Vault using the integration samples here: https://github.com/rajanadar/VaultSharp/blob/master/test/VaultSharp.Samples/HowToRunThisTestProgram.md
